### PR TITLE
Additional mpl 1.5.1 graphic test urls

### PR DESCRIPTION
**Pull request by bjlittle**
We have graphical test cases that perform more than one graphical test per test case.

These additional urls are for those outstanding multi-graphic test cases.

Also see https://github.com/SciTools/test-images-scitools/pull/6

### CODE REVIEW

**Comment by bjlittle** [comment]
Raise issue https://github.com/SciTools/iris/issues/2176 to capture the non-ideal situation of several graphical test cases calling `check_graphic` more than once.

File: lib/iris/tests/results/imagerepo.json [hunk 1]
```diff
@@ -146,14 +159,17 @@
...
     "iris.tests.test_analysis.TestRotatedPole.test_all.2": [
         "https://scitools.github.io/test-images-scitools/image_files/0f5c99da44a0f4640374961907f324a5dac767eb.png", 
-        "https://scitools.github.io/test-images-scitools/image_files/6b5e42fc27c146b1e5e6b6e57b7d4dce0891463b.png"
+        "https://scitools.github.io/test-images-scitools/image_files/6b5e42fc27c146b1e5e6b6e57b7d4dce0891463b.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/afae7fcc28956b67b89e39617e097fdca40045e3.png"
     ], 
     "iris.tests.test_analysis.TestRotatedPole.test_all.3": [
-        "https://scitools.github.io/test-images-scitools/image_files/27e9a9ca84937d9b13ba997cfbc6f461c925b20d.png"
+        "https://scitools.github.io/test-images-scitools/image_files/27e9a9ca84937d9b13ba997cfbc6f461c925b20d.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/2ce911bda2c1a416a57eeea6bf07728ed698f160.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: extent of the x axis has jumped a bit here, a bit concerning

File: lib/iris/tests/results/imagerepo.json [hunk 2]
```diff
@@ -568,19 +597,24 @@
...
     ], 
     "iris.tests.test_plot.TestPlotCoordinatesGiven.test_tx.3": [
-        "https://scitools.github.io/test-images-scitools/image_files/bc73ecda38e30c90eebce3b951d6f1d898484af8.png"
+        "https://scitools.github.io/test-images-scitools/image_files/bc73ecda38e30c90eebce3b951d6f1d898484af8.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/f1fb7eef7c66b42b67074a4ded9aafe4335b3804.png"
     ], 
     "iris.tests.test_plot.TestPlotCoordinatesGiven.test_tx.4": [
-        "https://scitools.github.io/test-images-scitools/image_files/39436a8c895057e3f29277858acfad6af89b7e36.png"
+        "https://scitools.github.io/test-images-scitools/image_files/39436a8c895057e3f29277858acfad6af89b7e36.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/6030ad045a05681b8a98eccdd617cab881d633cf.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: large extent jump :/

File: lib/iris/tests/results/imagerepo.json [hunk 3]
```diff
@@ -568,19 +597,24 @@
...
     ], 
     "iris.tests.test_plot.TestPlotCoordinatesGiven.test_tx.5": [
-        "https://scitools.github.io/test-images-scitools/image_files/5bdd0865ba9c5664dc1743f139a6838fc676393a.png"
+        "https://scitools.github.io/test-images-scitools/image_files/5bdd0865ba9c5664dc1743f139a6838fc676393a.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/b0f722c304de6473845b35a1dabdb82e0a99bf48.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: large y-extent jump :/

File: lib/iris/tests/results/imagerepo.json [hunk 4]
```diff
@@ -595,52 +629,64 @@
...
     ], 
     "iris.tests.test_plot.TestPlotCoordinatesGiven.test_yx.4": [
-        "https://scitools.github.io/test-images-scitools/image_files/d34377839f882f665fede8e867a496e20a7499d4.png"
+        "https://scitools.github.io/test-images-scitools/image_files/d34377839f882f665fede8e867a496e20a7499d4.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/fab55937cf3c5a94f4b9faa0c9f944fc980c29aa.png"
     ], 
     "iris.tests.test_plot.TestPlotCoordinatesGiven.test_yx.5": [
-        "https://scitools.github.io/test-images-scitools/image_files/57d78a1764a6944006f732e647452850259e34a4.png"
+        "https://scitools.github.io/test-images-scitools/image_files/57d78a1764a6944006f732e647452850259e34a4.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/88b4e9eb8512b81bcb7079a94bd86298e6d83657.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: jump in plot area, extent??

File: lib/iris/tests/results/imagerepo.json [hunk 5]
```diff
@@ -737,19 +788,24 @@
...
     ], 
     "iris.tests.test_quickplot.TestQuickplotCoordinatesGiven.test_tx.4": [
-        "https://scitools.github.io/test-images-scitools/image_files/d6d5f20c52a1caad2dee6a808106cf188085f3d6.png"
+        "https://scitools.github.io/test-images-scitools/image_files/d6d5f20c52a1caad2dee6a808106cf188085f3d6.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/ab44fe3502b6c7e4173f7c34cf6434977d993176.png"
     ], 
     "iris.tests.test_quickplot.TestQuickplotCoordinatesGiven.test_tx.5": [
-        "https://scitools.github.io/test-images-scitools/image_files/62a58743b114392de24ef212674c21f71bb4f372.png"
+        "https://scitools.github.io/test-images-scitools/image_files/62a58743b114392de24ef212674c21f71bb4f372.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/d97ff8b837e7acb6f233f4ba35e1eda8d57547d8.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: y-extent jump :/

File: lib/iris/tests/results/imagerepo.json [hunk 6]
```diff
@@ -764,38 +820,48 @@
...
     ], 
     "iris.tests.test_quickplot.TestQuickplotCoordinatesGiven.test_yx.4": [
-        "https://scitools.github.io/test-images-scitools/image_files/139739470587376aebae270abeca5100e2e996ff.png"
+        "https://scitools.github.io/test-images-scitools/image_files/139739470587376aebae270abeca5100e2e996ff.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/82737f30059330b4d1b8286110a482ff9e2975f1.png"
     ], 
     "iris.tests.test_quickplot.TestQuickplotCoordinatesGiven.test_yx.5": [
-        "https://scitools.github.io/test-images-scitools/image_files/6e2ee19aab2fdcdf3e3cb940987abcd1d682fe48.png"
+        "https://scitools.github.io/test-images-scitools/image_files/6e2ee19aab2fdcdf3e3cb940987abcd1d682fe48.png", 
+        "https://scitools.github.io/test-images-scitools/image_files/0bf0adf4e0c7998c6b5e879d2af1e9dec4e13359.png"
```
Comment (marqh) [inline lib/iris/tests/results/imagerepo.json]: extent jumps

**Comment by bjlittle** [comment]
@marqh Note that, of the above extent issue plots use `scatter` ...

**Comment by marqh** [comment]
a few of these images have extent shifts

> Note that, of the above extent issue plots use scatter ...

@bjlittle please may you raise a new Iris issue, collecting references to these tests and giving a bit of context to enable follow up of these activities independent of this PR

**Comment by marqh** [comment]
I think we should take these changes on and look into concerns separately, not hold back this PR 

I am minded to merge, pending travis success

**Comment by marqh** [comment]
> @bjlittle please may you raise a new Iris issue, collecting references to these tests and giving a bit of context to enable follow up of these activities independent of this PR

@bjlittle 
I am merging this, please ensure that this issue is raised and references back to here so we can track the acceptance of change

thank you

**Comment by bjlittle** [comment]
@marqh Raised issue https://github.com/SciTools/iris/issues/2178 to cover this mpl behaviour

**Pull request merged**